controllers/lists_controller.go

```diff
@@ -36,6 +36,11 @@ type ListJsonFormat struct {
 	ListOptionId int64
 }
 
+type AllListJsonFormat struct {
```

Comment:
__[gometalinter]__
* [golint] exported type AllListJsonFormat should have comment or be unexported
* [golint] type AllListJsonFormat should be AllListJSONFormat